doc/user/writing_spec.md

```diff
@@ -0,0 +1,45 @@
+# Writing Cluster Spec
+ 
+Users tell operator the desired state of an etcd cluster via spec in `Cluster` third party resource (TPR).
```

Comment:
The cluster spec is used to tell the operator ...

Let us avoid "we", "you", "users" when possible.